start/server/resolvers.js

```diff
@@ -17,8 +19,14 @@ const getAllBooks = () => {
   return books;
 };
 
-const addBook = (title, name) => {
-  books.push({ title: title, author: {name: name }});
+const addBook = (title, name, id) => {
+  books.push({ id: id, title: title, author: {name: name }});
+  console.log(books);
+  return true;
+};
+
+const deleteBook = (idToRemove) => {
+  books = books.filter((item) => item.id !== idToRemove);
```

Comment:
You may be aware of this, but it's usually best practice to not reassign a variable if we can: https://blog.sapegin.me/all/avoid-reassigning-variables/. 

```suggestion
  const filteredBooks = books.filter((item) => item.id !== idToRemove);
```